norman_objects/services/hug/huggingface_download_request.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime, timezone
+from typing_extensions import override
+
+from norman_objects.norman_base_model import NormanBaseModel
+from norman_objects.shared.messages.asset_message import AssetMessage
+from norman_objects.shared.messages.entity_type import EntityType
+from norman_objects.shared.status_flags.status_flag import StatusFlag
+from norman_objects.shared.status_flags.status_flag_name import StatusFlagName
+from norman_objects.shared.status_flags.status_flag_value import StatusFlagValue
+
+
+class HuggingFaceDownloadRequest(NormanBaseModel):
+    account_id: str
+    model_id: str
+    version_id: str
+    asset_id: str
+    asset_name: str
+    huggingface_model_id: str
```

Comment:
Naming: `huggingface_model_name`

1) ID and Name are used interchangeably in Huggingface
2) Aligns with our property ordering scheme (names after ids)
3) Reflects that the value is closer to a proper name than to an alphanumerical id